testing-and-tdd/jasmine-testing.md

```diff
@@ -331,55 +365,55 @@ function fizzBuzz(num) {
-    expect(fizzBuzz(30)).toBe("fizzbuzz");
-  });
+it("should return 'fizzbuzz' when given a multiple of 3 and 5", function(){
+expect(fizzBuzz(15)).toBe("fizzbuzz");
+expect(fizzBuzz(30)).toBe("fizzbuzz");
+});
 });
 </code></pre>
+
 </details>
 
 ### Independent Practice
 
 Exploring new matchers:
+
 1. Re-implement Challenge 6, "Sleeping In", from the basic JavaScript practice linked above, using TDD. Since this function returns a boolean, use the matchers `toBeTruthy` and `toBeFalsy` in your tests.
 2. Write a function `mySplit` that takes a string, and returns an array of its letters, using TDD. Use the matcher `toContain` in your test.
-    - e.g. `mySplit("dog") => ["d","o","g"]`
+   - e.g. `mySplit("dog") => ["d","o","g"]`
 
 ### Challenge
 
-*Challenge 1:* Read the docs on [`beforeEach`](https://jasmine.github.io/api/edge/global.html#beforeEach) and [`afterEach`](https://jasmine.github.io/api/edge/global.html#afterEach). Try to write a test that uses `beforeEach` to set up tests. There's a good example in the [Jasmine Tutorial](https://jasmine.github.io/tutorials/your_first_suite).
+_Challenge 1:_ Read the docs on [`beforeEach`](https://jasmine.github.io/api/edge/global.html#beforeEach) and [`afterEach`](https://jasmine.github.io/api/edge/global.html#afterEach). Try to write a test that uses `beforeEach` to set up tests. There's a good example in the [Jasmine Tutorial](https://jasmine.github.io/tutorials/your_first_suite).
 
-*Challenge 2:* You can also run Jasmine tests in the browser for a nicer UI! Follow the [installation instructions on Jasmine's github repo](https://github.com/jasmine/jasmine) to make your tests run in the browser instead of the command line.
+_Challenge 2:_ You can also run Jasmine tests in the browser for a nicer UI! Follow the [installation instructions on Jasmine's github repo](https://github.com/jasmine/jasmine) to make your tests run in the browser instead of the command line.
 
+### Supplemental Materials
 
-### Supplemental Materials 
 - [Automated Testing with Jasmine](https://atom-morgan.github.io/automated-testing/)
 - JS testing with Jasmine blog post, [link](https://automationpanda.com/2018/01/26/javascript-testing-with-jasmine/#project_structure)
-- Video by Dylan C. Israel [Unit Testing in JavaScript and Jasmine](https://www.youtube.com/watch?v=h2eWfvcAOTI) *starts at 12:00 minutes to show mocha test.*
-
+- Video by Dylan C. Israel [Unit Testing in JavaScript and Jasmine](https://www.youtube.com/watch?v=h2eWfvcAOTI) _starts at 12:00 minutes to show mocha test._
 
 ### Check for Understanding
 
-*Question:* What is Jasmine and how does it work in your project?
+_Question:_ What is Jasmine and how does it work in your project?
 
-*Exercise:* Each student should pick a matcher, like `toBeNull()`. Then describe that matcher to the class and how it should be used.
+_Exercise:_ Each student should pick a matcher, like `toBeNull()`. Then describe that matcher to the class and how it should be used.
```

Comment:
Explain me please...this file also contain same changes.

Reply:
Sorry, Actually I had some extensions installed in my vs code that unnecessarily make some changes as soon as I save the file. That extension prettifies the code but along with that changes '' to "" etc are also done.

Reply:
What would I have to do sir?  Just tell me and I will do that ASAP.

Reply:
Sir, may I make a new fresh PR?

Reply:
Sir I have made a new PR.

Reply:
Please fix this PR only, as making a new PR is not a good practice.

Reply:
I agree - on a real team, PRs with syntax fixes, etc are not appreciated unless the issue is specifically for fixing syntax & typos.  This is because anything not directly related to the issue is just noise, making it harder to see the real changes a teammate is trying to review.  As we are just starting, and I see you've made #1233 , I will close this. Next time however, please use this opportunity to learn enough about git that you can change the same branch the way you want.